docs/pages/versions/unversioned/index.mdx

```diff
@@ -50,6 +49,19 @@ The easiest way to create a bare React Native app with support for the Expo SDK
   href="/workflow/using-libraries"
 />
 
+## Using pre-release versions
+
+New Expo SDK versions are released three times each year. In between releases, we publish pre-release versions of the `expo` package and all of the Expo SDK packages. Pre-releases are not considered to be stable and should only be used if you are comfortable with the risk of encountering bugs or other issues.
+
+Pre-release versions of the `expo` package are versioned with an `-alpha` suffix, such as `50.0.0-alpha.1`. To install, for example, SDK 50 alpha, you can run the following command:
+
+<Terminal
+  cmd={['# Install the alpha version of expo and its related packages', '$ npm install expo@50.0.0-alpha && npx expo install --fix']}
```

Comment:
we will want to update this once we come up with a better process and having tooling that supports that. for example, we should probably tag alpha releases with the `alpha` tag on npm. cc @wschurman